src/App.js

```diff
@@ -123,6 +124,11 @@ const App = ({ locale, location, dependencies: { appSessionRef, sessionManager }
                 exact
                 component={PlanCalculator}
               />
+              <PrivateRoute
+                path={'/checkout-process/:pathType/:planType?'}
```

Comment:
How about this simply being:
```suggestion
                path={'/checkout/:pathType/:planType?'}
```

Reply:
Yes, how I will rename the component. My idea is to change the url too.